.github/workflows/pullrequestforrelease.yml

```diff
@@ -0,0 +1,33 @@
+name: Raise a Pull Request
+
+on:
+  push:
+    branches:
+      - "PR"  # Support wildcard matching
+
+jobs:
+  pull-request:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Debug information
+        run: |
+          echo "Current directory: $(pwd)"
+          echo "GitHub Token: ${{ secrets.GITHUB_TOKEN }}"
+          echo "Source Branch: ${{ github.ref }}"
+          echo "Destination Branch: master"
+
+      - name: Raise pull request
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: "PR"                                 # If blank, default: triggered branch
+          destination_branch: "master" # If blank, default: master
+          pr_label: "automation" 
+          pr_title: "OV8.4 Validation" # Title of pull request
```

Comment:
Can we given generic PR title like 'OVmaj_ver.min_ver validation' in title

Reply:
Yes can be done , will make the changes

Reply:
Have modified the file to take the release version from the jenkins conf file and then use the same to create PR and branch without hardcoding any values.